Files/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -54,10 +54,22 @@
                 </ResourceDictionary>
             </CommandBar.Resources>
             <CommandBar.PrimaryCommands>
+                <AppBarButton
+                    x:Name="EmptyRecycleBinButton"
+                    x:Uid="InnerEmptyRecycleBinButton"
+                    Label="Empty"
+                    IsEnabled="{x:Bind ViewModel.CanEmptyRecycleBin, Mode=OneWay}"
+                    Command="{x:Bind ViewModel.EmptyRecycleBinCommand, Mode=OneWay}"
+                    Visibility="{x:Bind ViewModel.InstanceViewModel.IsPageTypeRecycleBin, Mode=OneWay}">
+                    <AppBarButton.Icon>
+                        <FontIcon Glyph="&#xEF88;" FontFamily="{StaticResource RecycleBinIcons}" />
+                    </AppBarButton.Icon>
+                </AppBarButton>
                 <AppBarButton
                     x:Name="NewEmptySpaceAppBarButton"
                     x:Uid="BaseLayoutContextFlyoutNew"
                     IsEnabled="{x:Bind ViewModel.InstanceViewModel.IsCreateButtonEnabledInPage, Mode=OneWay}"
+                    Visibility="{x:Bind ViewModel.InstanceViewModel.IsPageTypeRecycleBin, Mode=OneWay, Converter={StaticResource NegatedBoolToVisibilityConverter}}"
```

Comment:
```suggestion
```
I think it's best to keep this visible to ensure consistency with other directories.

Reply:
@yaichenbaum The other changes look ok. But why display the New button if it is not in use? It is neither useful nor aesthetic. I suggest creating a CanCreateNew property, temporarily true if the page is not Recycle Bin. This will allow you to modify its definition for other pages later.